configs/prod/roles/insights.json

```diff
@@ -15,6 +15,20 @@
           "permission": "inventory:*:read"
         }
       ]
+    },
+    {
+          "name": "Advisor Viewer",
```

Comment:
👍  I'm curious so this permission and role will only exist on prod?

Reply:
now I understand 😂  https://github.com/RedHatInsights/rbac-config/pull/509

Reply:
yeah.. I split the change, because long time ago was told me to make separeted PR for prod and stage, so, if there's an issue on deployment It's will be more easy to revert only one..